js/languages/en-US.json

```diff
@@ -87,6 +87,57 @@
     "userLoading": "Loading...",
     "userNotFound": "Information for %{guid} was not found on the network."
   },
+  "about": {
+    "linkText": "About OpenBazaar",
+    "brand": "OpenBazaar",
+    "blurbs": {
+      "tradeFree": "OpenBazaar is made possible by an international community of developers and volunteers contributing their time to help make trade free."
+    },
+    "version": "version",
+    "versionCode": "2.0",
+    "client": "Client",
+    "server": "Server",
+    "btnUpdateCheck": "Check for Update",
+    "menu": {
+      "sectionHeader": "MENU",
+      "btnNext": "Next",
+      "btnPrevious": "Previous"
```

Comment:
Okay.

This was related to our discussion about the menu not scrolling as the content pane scrolled. IIRC it was @rmisio who first considered what would happen if the menu was so long it would also need to scroll. Thinking about that I considered a menu that could be paged forward and back using Next and Previous buttons and not scrolled.